### PR TITLE
spring-cloud-commons: initial integration

### PR DESCRIPTION
**Pull request by onionpsy**

### CODE REVIEW

File: projects/spring-cloud-commons/build.sh [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+# limitations under the License.
+#
+################################################################################
+
+mv $SRC/*.dict $OUT
+
+mkdir -p $JAVA_HOME
+cp -rL "/usr/lib/jvm/java-17-openjdk-amd64/." "$JAVA_HOME" || true
+
+cat > patch.diff <<- EOM
```
Comment (oliverchang) [inline projects/spring-cloud-commons/build.sh:23]: Can you add a comment here for why this is needed?

**Review by oliverchang — APPROVED** [review]
thanks!

File: projects/spring-cloud-commons/build.sh (hunk 1 above)
Comment (oliverchang) [reply projects/spring-cloud-commons/build.sh:23]: friendly ping  @onionpsy

Comment (onionpsy) [reply projects/spring-cloud-commons/build.sh:24]: @oliverchang I opted for a cleaner solution :). Sorry for the delay

Comment (oliverchang) [reply projects/spring-cloud-commons/build.sh:24]: Did you push that change? I still see the patch in this PR.

I'll merge it for now, but please address this in another PR.

**Pull request merged**